### ISSUE TITLE
How to send the first view? [SSR][bootstrap=false]

### ISSUE THREAD

**Issue by sneko** [MatteoGabriele/vue-gtag#193]
Hi @MatteoGabriele ,

I'm struggling to send the first view when reaching my website.

All following `$router.push` will be catched by `vue-gtag` (I can see them inside GTM Debug Chrome extension), but the first one is not. I'm wondering if it could come from the fact I use server-side rendering, however the script seems loaded when I call `bootstrap()`:
![image](https://user-images.githubusercontent.com/8195958/92715126-5910c200-f35d-11ea-99f4-7c1584a650a4.png)

I also tried to set `bootstrap: true` so there is an immediate activation, but that's the same.

I'm not using Nuxt.js but the "raw" Vue SSR (often with `entry-client.js` and `entry-server.js`), if you have any idea what could be the cause, it would be great 👍 

Thank you,

EDIT: It seems when doing
```
      params: {
        send_page_view: true,
      },
```
the library catches the first one too. I knew it was in the docs but when trying earlier it was not working hmmm... Don't know what could have be in conflict (cache or other?). It catches in a different format (`http://localhost/home` instead of `/home` for the following page changes), but that's maybe the normal behavior.

**Comment by sneko** [comment]
It don't know if that's a bug with the GTM Debug extension, but having this first page in the "Hits" section is like random. I have 9/10 chance it appears, but sometimes it's not... so I'm not sure it works for my end users when in production hmmmm.

**Comment by MatteoGabriele** [comment]
usually for single page applications the first config sends a send_page_view false, so you might want to pass that true in the plugin config params, when you install the plugin.
I am not very much into SSR much, so might be that it needs to be different: instead of a first hit with send_page_view false you want that true and instead of sending event page_view u need a config hit for your page tracking, but I guess an SSR app is still kinda an SPA....right? :D

**Comment by MatteoGabriele** [comment]
you should check your `dataLayer` object in the window object to see what's in the array maybe? that is the most accurate to check

**Comment by kierans** [comment]
I'm having a related problem. The host page for my app (including the `<script>` include for the Gtags library) is rendered serverside. I have a `window.onload` event handler that tells my Vue app to bootstrap the analytics module which in turn calls `bootstrap` in vue-gtag.

I've noticed that even with `send_page_view: true` in my params the first view event isn't being sent to GA. I've checked the `dataLayer` object and I can't see any network traffic.

Debugging it with some `console.log` statements in a copy of `vue-gtag@1.10.0` I can see that when the [Routers `onReady` handler is invoked](https://github.com/MatteoGabriele/vue-gtag/blob/69a973bde60b32c30aeb095a8a69973c33e95731/src/page-tracker.js#L73) the value of `current` is `undefined`. Thus no data can be sent.

I don't know if it's a fault of the Vue Router (`vue@^2.6.10`, `vue-router@^3.1.6`) of if that's expected behaviour. Is there a way to check for a current route for the base where vue-gtags is bootstrapped post app initialisation?
